src/utils/validation/emailValidation.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Don't disable `import/prefer-default-export`

Reply:
I have to disable it